secure_mail/management/commands/email_signing_key.py

```diff
@@ -70,8 +70,11 @@ def handle(self, *labels, **options):
         if options.get('generate') and labels:
             raise CommandError("You cannot specify fingerprints and "
                                "--generate when running this command")
-
+        if options.get('generate') and not SIGNING_KEY_DATA["Passphrase"])
```

Comment:
```suggestion
        if options.get('generate') and not SIGNING_KEY_DATA["passphrase"])
```

NAK, this will not work as written. The "parameter" to GPG is (capitalized) "Passphrase", but the keyword argument must be all lowercase. See the [GPG docs on Generating Keys](https://pythonhosted.org/python-gnupg/#generating-keys) for more information.